src/enum/ImageTypes.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Shohjahon\RentSrc\enum;
+
+enum ImageTypes: string
+{
+    case DEFAULT = 'default.png';
```

Comment:
Enums are great, but ImageTypes makes me think that it contains only types and nothing else. Apparently, I'm getting default.png as a type. Or you should have rename it to something more verbose or add some explaining comments. Because I was expecting extensions like .png, .jpg etc.